<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilteringWithCVClass</class>
 <widget class="QWidget" name="FilteringWithCVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilteringWithCV</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>显示为直方图</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="pGraphicsViewer" name="graphicsView"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Img</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>去噪滤波</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>中值滤波</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>高斯滤波</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>均值滤波</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Gray</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action123">
   <property name="text">
    <string>123</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>pGraphicsViewer</class>
   <extends>QGraphicsView</extends>
   <header>../GraphicsViewer/pGraphicsViewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="FilteringWithCV.qrc"/>
 </resources>
 <connections/>
</ui>
